src/sentry/static/sentry/app/components/group/sentryAppExternalIssueForm.tsx

```diff
@@ -13,22 +13,40 @@ import getStacktraceBody from 'app/utils/getStacktraceBody';
 import withApi from 'app/utils/withApi';
 import {Client} from 'app/api';
 import {Group, PlatformExternalIssue, Event, SentryAppInstallation} from 'app/types';
-import {Field} from 'app/views/settings/components/forms/type';
+import {Field, FieldValue} from 'app/views/settings/components/forms/type';
+import FormModel from 'app/views/settings/components/forms/model';
+import {replaceAtArrayIndex} from 'app/utils/replaceAtArrayIndex.tsx';
```

Comment:
hmm do we not have a lint rule for this?

```suggestion
import {replaceAtArrayIndex} from 'app/utils/replaceAtArrayIndex';
```

Reply:
@billyvg I guess not